tools/coverage/Makefile

```diff
@@ -5,9 +5,20 @@ REGISTRY ?= gcr.io
 PROJECT  ?= knative-tests
 PUSH     ?= docker push
 
-coverage-image:
+# Making dev version of coverage-image
+coverage-image-dev:
+	@echo "Making dev version..."
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o coverage
 	chmod +x coverage
 	docker build -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:$(TAG)" -t "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev" .
 	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:$(TAG)"
-	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev"
\ No newline at end of file
+	$(PUSH) "$(REGISTRY)/$(PROJECT)/test-infra/coverage-dev:latest-dev"
+
+# Making production version of coverage-image
+production:
```

Comment:
can u keep the naming patter consistent with dev image? i.e. coverage-image-prod?